test/e2e/kubectl/kubectl.go

```diff
@@ -2069,6 +2069,103 @@ metadata:
 			gomega.Expect(outBuiltIn).To(gomega.Equal(outStatusSubresource))
 		})
 	})
+
+	ginkgo.Describe("Kubectl rollout", func() {
+		deploymentYaml := `
+		{
+			"apiVersion": "apps/v1",
+			"kind": "Deployment",
+			"metadata": {
+			  "name": "nginx-deployment"
+			},
+			"spec": {
+			  "replicas": 1,
+			  "selector": {
+				"matchLabels": {
+				  "test": "rollout"
+				}
+			  },
+			  "template": {
+				"metadata": {
+				  "labels": {
+					"test": "rollout"
+				  }
+				},
+				"spec": {
+				  "containers": [
+					{
+					  "name": "nginx-container",
+					  "image": "nginx:latest"
+					}
+				  ]
+				}
+			  }
+			}
+		}	  
+`
+		ginkgo.AfterEach(func() {
+			cleanupKubectlInputs(deploymentYaml, ns, "test=rollout")
+		})
+		ginkgo.It("update deployment env and rollback", func(ctx context.Context) {
+			// create deployment
+			if _, err := e2ekubectl.RunKubectlInput(ns, deploymentYaml, "create", "-f", "-"); err != nil {
+				framework.Failf("kubectl failed to create deployment")
+			}
+			if err := wait.PollUntilContextCancel(ctx, 10*time.Second, true, func(ctx context.Context) (done bool, err error) {
```

Comment:
this is a too short timeout, which would be easily exceeded when the node needs to pull the image. Can you please increase it, or use `WaitForDeploymentComplete` or other helper function instead?